spark/src/test/scala/org/apache/spark/sql/delta/typewidening/TypeWideningMetadataSuite.scala

```diff
@@ -49,8 +50,9 @@ trait TypeWideningMetadataTests extends QueryTest with DeltaSQLCommandTest {
 
   override protected def beforeAll(): Unit = {
     super.beforeAll()
-    sql(s"CREATE TABLE $testTableName (a int) USING delta " +
-      s"TBLPROPERTIES ('${DeltaConfigs.ENABLE_TYPE_WIDENING.key}' = 'true')")
+    sql(s"CREATE TABLE $testTableName (a int) USING delta TBLPROPERTIES (" +
+      s"'${DeltaConfigs.ENABLE_TYPE_WIDENING.key}' = 'true', " +
+      s"'${propertyKey(TypeWideningTableFeature)}' = 'supported')")
```

Comment:
Why is this necessary? Isn't the feature automatically enabled when setting the property?

Reply:
The *preview* feature is enabled by default, this forces the stable to be added instead.

I added a comment to clarify that and that we're testing with stable feature by default and only explicit set the preview feature in specific tests that cover the old behavior re: `tableVersion`.
+ added a missing test for the preview feature